yt_dlp/extractor/gofile.py

```diff
@@ -0,0 +1,57 @@
+# coding: utf-8
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError
+)
+
+
+class GofileIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?gofile\.io/d/(?P<id>[^/]+)'
+    _TESTS = [{
+        'url': 'https://gofile.io/d/AMZyDw',
+        'md5': '57c04f65556ddbecf9697528274c3212',
+        'info_dict': {
+            'id': 'de571ac1-5edc-42e2-8ec2-bdac83ad4a31',
+            'filesize': 928116,
+            'ext': 'mp4',
+            'title': 'nuuh.mp4'
+        }
+    }]
+
+    def _real_extract(self, url):
+        file_id = self._match_id(url)
+
+        # Create guest account
+        accountdata = self._download_json_handle('https://api.gofile.io/createAccount', 'Gofile', note='Getting a new guest account')
+        status = accountdata[0]['status']
+        accountToken = accountdata[0]['data']['token']
+        print("[Gofile] Status:", status, "accountToken:", accountToken)
```

Comment:
Don't use print
There's to_screen func in common.py afair but more importantly printing this to screen is not needed imo